Misc/sbom.spdx.json

```
@@ -1643,19 +1643,19 @@
           "checksumValue": "988a74f5fbb59baca2d54e41447997ada92f4ebc59888dfb717438013f859117"
```

Ah ha, that's why :)

```suggestion
          "checksumValue": "935ae51d0ff0bf1403f0ecc1ff02b8f685d09053618558c07fbe4bd2abbc5dd1"
```